### ISSUE TITLE
@doc strings folding doesn't work using the Collapse doc comments actions

### ISSUE THREAD

**Issue by kandeshvari** [KronicDeth/intellij-elixir#937]
I know two ways to collapse docstrings:

- manually: `ctrl + shift + a` -> `collapse doc comments`
- automatically: `ctrl + shift + a` -> `Collapse by default` -> activate Documentation strings

neither of two doesn't work. Is `@doc` marks internally as a docstring?

**Comment by KronicDeth** [comment]
Nah, they use the normal code folding: `Cmd + +` and `Cmd + -` to unfold`.  There's also the +/- folding icons in the gutter.

![code folding doc](https://user-images.githubusercontent.com/298259/33525916-d91664f6-d7fd-11e7-8b1c-b6657eb37adc.gif)

I can look it it's possible to get them to work with the documentation folding.  It might not.  `@doc` and `@moduledoc` aren't keywords and aren't parsed special by the lexer or parser.  The coloration difference is due to an annotator triggered by the module attribute name.

The closest current setting to folding `@doc` and `@moduledoc` by default is

1. `Preferences > Editor > General > Code Folding`
2. Check "Elixir module attributes"

but that will fold all module attributes, not just the `*doc` ones.

**Comment by silva96** [comment]
checking `Elixir module attributes` under `Preferences > Editor > General > Code Folding` is not working for me.
